components/GlobalStyles.js

```diff
@@ -1,11 +1,16 @@
 import { createGlobalStyle } from "styled-components";
 
 const GlobalStyles = createGlobalStyle`
+    :root {
+        --text-primary: black;
+    }
+
     html,
     body {
         padding: 0;
         margin: 0;
         font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Helvetica, Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol";
+        max-width: 375px;
```

Comment:
If you wanna use `max-width`, I would recommend to add this to your `body` or `main`element instead of the `html` element. 
Additionally, a `max-width` of 375px seems pretty restrictive. As we want to develop a responsive app, I suggest to choose a less restrictive width. 
Later on, you can also use media-queries to make your app more responsive.